### DISCUSSION TITLE
A question on terminology: CCPP-Framework, CCPP-Physics

### DISCUSSION THREAD

**Discussion by climbfuji** [NCAR/ccpp-doc#52] [Q&A] [answered]
We use `CCPP-Framework` and `CCPP-Physics` in the technical documentation for the two components, and refer to them as `[the] CCPP-Framework is ...` and `[the] CCPP-Physics is ...`.

While this is all good for the framework, it seems weird for the physics, because "physics" is plural? Shouldn't we use "CCPP-Physics are ..."? It would be different if it was called "the CCPP-Physics library" but we are not using this terminology (I would prefer that).

**Comment by ligiabernardet** [comment, marked as answer]
In my view, it is fine to use "physics" as singular. A quick Google search
corroborates that, e.g., *Physics, while plural in derivation, is generally
construed as a singular noun except when referring to several “physics”
(i.e., two or more different systems ... *(from
https://cementanswers.com/is-physics-singular-or-plural/).

But on another note, I suggest standardizing as *CCPP** Physics* (and not
CCPP-Physics) and *CCPP Framework* (and not CCPP-Framework) as we have
been. using in other contexts.

**Reply by climbfuji** [reply]
Sounds good to me. Yes, I noticed that the tech doc uses CCPP-Framework and CCPP-Physics with dashes in between.
